database/schema/12-wasm.sql

```diff
@@ -0,0 +1,56 @@
+CREATE TYPE ACCESS_CONFIG AS
+(
+    permission  INT,
+    address     TEXT
+);
+
+CREATE TABLE wasm_params
+(
+    one_row_id                      BOOLEAN         NOT NULL DEFAULT TRUE PRIMARY KEY,
+    code_upload_access              ACCESS_CONFIG   NOT NULL,
+    instantiate_default_permission  INT             NOT NULL,
+    max_wasm_code_size              INTEGER         NOT NULL,
+    height                          BIGINT          NOT NULL
+);
+
+
+CREATE TABLE wasm_code
+(
+    sender                  TEXT            NULL,
+    byte_code               BYTEA           NOT NULL,
+    instantiate_permission  ACCESS_CONFIG   NULL,
+    code_id                 BIGINT          NOT NULL UNIQUE,
+    height                  BIGINT          NOT NULL
+);
+CREATE INDEX wasm_code_height_index ON wasm_code (height);
+
+CREATE TABLE wasm_contract
+(
+    sender                  TEXT            NULL,
```

Comment:
Is `sender` value usually `NULL` or should we also add reference to account?

Reply:
It's null in genesis contracts but should not be empty in general..